src/Login.tsx

```diff
@@ -49,41 +52,43 @@ const Login: React.FC = () => {
   }
 
   return (
-    <AuthContext.Consumer>
```

Comment:
Removing this stuff is incidental cleanup.  When I first wrote `Login` I didn't know about the `useContext` hook, so I thought the only way for me to have access to `setToken` (which is available here via the `AuthContext`) was to do this `<AuthContext.Consumer>` component nonsense.  Much more intelligible this way!